### DISCUSSION TITLE
Github Explore Feedback

### DISCUSSION THREAD

**Discussion by jeffcampbellmakesgames** [community/community#22114] [New to GitHub]
<p>At one point GitHub’s Explore area was an interesting place to check in on an discover new projects related to topics I was interested in or related to projects I support/contribute to. Nowadays all I see are:</p>
<p>* Ads for services or conferences I don’t care about, sometimes taking up over half the page’s content.</p>
<p>* A ton of javascript/node projects I don’t have any interest in seeing and aren’t related to topics I’ve starred or projects I’ve supported.</p>
<p>* Recently, a ton of Github profiles for people whose work focus is on topics I don’t have any interest in seeing and aren’t related to topics I’ve starred or projects I’ve supported.</p>
<p>I don’t see any way to filter this content and its disappointing because there was a time when the focus was solely on projects and it was really a great way of discovering them. Now I’d say about 5-10% of the content is interesting or relevant.</p>

**Comment by ghost** [comment, marked as answer]
<p>Hi <a class="mention" href="/jeffcampbellmakesgames">@jeffcampbellmakesgames</a> - thanks for taking the time to reach out in our GitHub community. The best way to provide feedback is to submit this through our <a href="https://support.github.com/contact/feedback">product feedback form</a> - which gets logged with our product team. While we cannot respond to all feedback submissions, we do consider every request and really appreciate our users taking the time to give us feedback.</p>
